src/common/models/measure/measure.ts

```diff
@@ -28,7 +28,7 @@ export interface MeasureValue {
   title?: string;
   units?: string;
   formula?: string;
-  format?: string;
+  format?: {} | string;
```

Comment:
Measure probably should have format as discriminated union, so we don't need to use `typeof` in runtime

Reply:
Object is not a primitive and thus will not be serialised-deserialised properly. You should add test for this case